src/components/Text.js

```diff
@@ -166,3 +166,34 @@ export const BoldText = styled(RegularText)`
       font-family: bold-italic;
     `};
 `;
+
+export const HeadlineText = styled(RegularText)`
+  font-weight: 700;
+  font-family: bold;
+
+  ${(props) =>
+    props.medium &&
+    css`
+      font-size: ${normalize(18)};
+      line-height: ${normalize(23)};
+    `};
+
+  ${(props) =>
+    props.small &&
+    css`
+      font-size: ${normalize(16)};
+      line-height: ${normalize(21)};
+    `};
+`;
+
+export const CategoryText = styled(RegularText)`
+  font-weight: 700;
+  font-family: bold;
```

Comment:
if these should be bold, they can be built on top of `BoldText` instead of `RegularText`. and i guess 700 is not needed in the master repo. isn't it a release specific thing? the `BoldText` did not have any font-weight.

Reply:
Yes, you're absolutely right. I also think CategoryText is too much for the main repository and I will remove it.

Reply:
https://github.com/smart-village-solutions/smart-village-app-app/pull/920/commits/24147097e638fbb4faab803fab473fc7bbe10503